src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/TcpTransportSecurityElement.cs

```diff
@@ -36,5 +36,18 @@ public TcpClientCredentialType ClientCredentialType
         //    get { return (SslProtocols)base[ConfigurationStrings.SslProtocols]; }
         //    private set { base[ConfigurationStrings.SslProtocols] = value; }
         //}
+
+        internal void ApplyConfiguration(TcpTransportSecurity security)
+        {
```

Comment:
We do have support for SslProtocols in the implementation. Can you uncomment those pieces?